web/src/app/wizard/WizardRouter.js

```diff
@@ -239,8 +240,8 @@ export default class WizardRouter extends React.PureComponent {
     )
   }
 
-  renderSubmittedContent() {
-    const { complete, errorMessage, value } = this.state
+  function renderSubmittedContent(state) {
```

Comment:
This is giving me the error `renderSubmittedContent' was used before it was defined`. You can bring this whole function into the component's body, and then you can delete the `state` parameter here since it will be in scope.